x/rollapp/keeper/rollapp.go

```diff
@@ -245,3 +245,109 @@ func (k Keeper) IsRollappStarted(ctx sdk.Context, rollappId string) bool {
 	_, found := k.GetLatestStateInfoIndex(ctx, rollappId)
 	return found
 }
+
+func (k Keeper) MarkRollappAsVulnerable(ctx sdk.Context, rollappId string) error {
+	return k.FreezeRollapp(ctx, rollappId)
+}
```

Comment:
bit weird since vulernable is not synonymous with vulnerable

IMO we do not need to explicitly mark them as vulnerable
They are vulnerable implicitly when their DRS ver is a vulnerable ver

So remove this function and go directly to freeze

Reply:
ahh, you did this in case more logic gets added or something right?